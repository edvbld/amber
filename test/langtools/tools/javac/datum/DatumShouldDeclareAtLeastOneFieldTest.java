/*
 * @test /nodynamiccopyright/
 * @summary smoke negative test for datum classes
 * @compile/fail/ref=DatumShouldDeclareAtLeastOneFieldTest.out -XDrawDiagnostics DatumShouldDeclareAtLeastOneFieldTest.java
 */

public class DatumShouldDeclareAtLeastOneFieldTest {
    static abstract __datum D1() { }

    static __datum D2() { }
}
