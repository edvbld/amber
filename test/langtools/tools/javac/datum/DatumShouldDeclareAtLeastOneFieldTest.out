DatumShouldDeclareAtLeastOneFieldTest.java:8:34: compiler.err.datum.must.declare.at.least.one.field
DatumShouldDeclareAtLeastOneFieldTest.java:10:25: compiler.err.datum.must.declare.at.least.one.field
2 errors
